src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -507,18 +508,20 @@
                     </Button>
                 </StackPanel>
             </Border>
-            <Border x:Name="TracePointer"
-                    Width="16"
-                    Height="18"
-                    Margin="48,50,0,0"
-                    HorizontalAlignment="Left"
-                    VerticalAlignment="Top"
-                    Visibility="Collapsed">
-                <FontIcon Foreground="Red"
-                          FontFamily="{StaticResource CalculatorFontFamily}"
-                          FontSize="18"
-                          Glyph="&#xE3B3;"/>
-            </Border>
+            <Grid x:Name="TracePointer"
+                  HorizontalAlignment="Left"
+                  VerticalAlignment="Top"
+                  Visibility="Collapsed">
+                <Border x:Name="CursorShadow"/>
+                <Path x:Name="CursorPath"
+                      Width="18"
+                      Height="18"
+                      Fill="White"
```

Comment:
I assume this is fine in high contrast since it is the same as mouse color?

Reply:
Well, I was thinking 2 fold...  1 is that this is somewhat temporary as we'll be changing the color to match the line as a future update, and 2, white fill with black stroke will always provide contrast.